docs/developer-guide.md

```diff
@@ -0,0 +1,88 @@
+# Developers' guide
+
+This page is a developer's guide to popular commands used in the process of setting up environment and testing.
+
+## Basic pipenv knowledge
+
+The general purpose of pipenv is similar to that of Package installer for Python (PIP) and built-in venv.
```

Comment:
link the official docs https://pipenv.pypa.io/en/latest/

Reply:
Link at the end of the paragraph